base/float.jl

```diff
@@ -631,6 +631,7 @@ Test whether a number is infinite.
 See also: [`Inf`](@ref), [`iszero`](@ref), [`isfinite`](@ref), [`isnan`](@ref).
 """
 isinf(x::Real) = !isnan(x) & !isfinite(x)
+isinf(x::IEEEFloat) = abs(x) == oftype(x, Inf)
```

Comment:
I see even more improvement from `===` instead of `==`

Reply:
I'm not seeing a big difference but it does appear to be very slightly better. I'll change it.